src/editor/components/modals/reversion-modal.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Modal from "../../../shared/components/modal";
+import ModalTitle from "../../../shared/components/modal-title";
+import ModalContent from "../../../shared/components/modal-content";
+import ModalCall from "../../../shared/components/modal-call";
+import { TextButton } from "../../../shared/components/buttons";
+
+export default class DeleteModal extends React.Component {
+  static propTypes = {
+    onCloseOrCancel: PropTypes.func.isRequired,
+    onRevert: PropTypes.func.isRequired,
+    visible: PropTypes.bool,
+    aboveOtherModals: PropTypes.bool
+  };
+
+  render() {
+    return (
+      <Modal
+        visible={this.props.visible}
+        onCloseOrCancel={this.props.onCloseOrCancel}
+        aboveOtherModals={this.props.aboveOtherModals}
+      >
+        <ModalTitle>Revert to this revision</ModalTitle>
+        <ModalContent>This action cannot be undone.</ModalContent>
```

Comment:
This actually isn't technically true. :) You can always revert back to what was there before. I'd recommend using something similar to what Google docs uses as outlined in the orginal issue:

https://github.com/iodide-project/iodide/issues/2047#issue-469355732

Something like:

```jsx
<ModalTitle>Restore this revision?</ModalTitle>
<ModalContent>This will restore your notebook to version from 16 July at 14:53</ModalContent>
```

Reply:
😸

Date is now displayed.